src/CustomMarshalers/src/System/Runtime/InteropServices/CustomMarshalers/IDispatch.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+
+namespace System.Runtime.InteropServices.CustomMarshalers
+{
+    [ComImport]
+    [Guid("00020400-0000-0000-C000-000000000046")]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    internal interface IDispatch
```

Comment:
We have an IDispatch defined in corelib in src\System\Runtime\InteropServices\NativeMethods.cs, with the same visibility and attributes, but without members.  Should we consolidate this into a common file that's shared in Common between the repos?

Reply:
I thought that was private? I guess we could expose it. Since the type is well-defined I don't see a lot of concern with the duplication, but won't argue if it is desired.

Reply:
I wasn't suggesting exposing it.  Just that we could define it in a file once and include that file in both projects, rather than duplicating the definition in two different files.